front/src/modules/simulationResult/components/SpaceTimeChart/drawTrain.ts

```diff
@@ -62,33 +48,17 @@ export default function drawTrain(
     d3.select(releventLine).attr('transform', `translate(${translation})`);
   };
 
-  let debounceTimeoutId: number;
-
-  function debounceUpdateDepartureArrivalTimes(
-    computedDepartureArrivalTimes: TrainsWithArrivalAndDepartureTimes[],
-    interval: number
-  ) {
-    clearTimeout(debounceTimeoutId);
-    debounceTimeoutId = window.setTimeout(() => {
-      dispatchUpdateDepartureArrivalTimes(computedDepartureArrivalTimes);
-    }, interval);
-  }
-
   const drag = d3drag()
     .container((d) => d as DragContainerElement) // the component is dragged from its initial position
     .on('end', () => {
       dragTimeOffset(dragFullOffset);
-      dispatchUpdateMustRedraw(true);
     })
     .on('start', () => {
       dragFullOffset = 0;
       dispatchUpdateSelectedTrainId(trainToDraw.id);
     })
     .on('drag', (event) => {
       dragFullOffset += rotate ? event.dy : event.dx;
-      const offset = getDragOffsetValue(dragFullOffset);
-      const newDepartureArrivalTimes = makeTrainList(trainSimulations, trainToDraw.id, offset);
```

Comment:
If we don't need `makeTrainList()` anymore, it should be deleted from ChartHelpers.ts (it was the only component where it was used).